src/storage-manager/README.md

```diff
@@ -0,0 +1,122 @@
+## Storage Manager
```

Comment:
Can you add some description about the purpose of storage manager? I'm not quite understand when should I deploy this service.

Reply:
Done